packages/php-table-backend-utils/README.md

```diff
@@ -244,6 +244,20 @@ SNOWFLAKE_DATABASE=KEBOOLA_CI_TABLE_UTILS
 SNOWFLAKE_WAREHOUSE=
 ```
 
+##### Setup snowflake Key-pair authentication
+For the new key-pair authentication, which will be the only preferred method going forward, please follow these steps:
+1. Open a terminal window and generate a private key and a public key using the following command:
+```bash
+openssl genrsa 2048 | openssl pkcs8 -topk8 -inform PEM -out rsa_key.p8 -nocrypt
+openssl rsa -in rsa_key.p8 -pubout -out rsa_key.pub
+```
+2. Assign the public key to a Snowflake user using the following command:
+```sql
+ALTER USER "KEBOOLA_CI_TABLE_UTILS" SET RSA_PUBLIC_KEY='MIIBIjANBgkqh...';
```

Comment:
ten public key taky musíš trimovat to je nuda
```
PUBLIC_KEY=$(sed '1d;$d' rsa_key.pub | tr -d '\n')
echo "ALTER USER \"KEBOOLA_CI_TABLE_UTILS\" SET RSA_PUBLIC_KEY='${PUBLIC_KEY}';"
```

Reply:
Dik pridal som to do toho examplu